e2e/nomostest/nt.go

```diff
@@ -615,19 +609,18 @@ func (nt *NT) portForwardGitServer() {
 		// re-init all repos
 		InitGitRepos(nt, allRepos...)
 		// attempt to recover by re-pushing the local repo states
-		for nn, remoteRepo := range nt.RemoteRepositories {
-			remoteURL := nt.GitProvider.RemoteURL(newPort, nn.String())
-			remoteRepo.pushAllToRemote(remoteURL)
-			remoteRepo.RemoteURL = remoteURL
+		for _, remoteRepo := range nt.RemoteRepositories {
+			remoteRepo.pushAllToRemote()
 		}
 		prevPodName = podName
 	}
-	nt.gitRepoPortForwarder = nt.newPortForwarder(
+	gitRepoPortForwarder := nt.newPortForwarder(
```

Comment:
If you're only gonna use this var on the next line, you can just return it here and save an allocation.

Reply:
done